testplan/runnable/interactive/base.py

```diff
@@ -123,7 +123,8 @@ def teardown(self):
 
         self._pool = None
         self._http_handler = None
-        self.target._close_file_logger()
+        if self.target.active:
```

Comment:
When teardown is called, active might already be False.

Reply:
I removed this condition, because now the test runner wait for the interactive handler thread to quit, so at this time the target (test runner) must be active.